mesa_geo/geoexperimental/jupyter_viz.py

```diff
@@ -0,0 +1,410 @@
+import sys
```

Comment:
Forking the entire file here is not sustainable. Needs to do it the way it is done here https://github.com/projectmesa/mesa-examples/blob/1933cbc8251ca047273e0f88193f692ccbb86de6/examples/sugarscape_g1mt/app.py#L9.